<launch>

  <node name="mav_shortest_path_planning"
    pkg="mav_coverage_planning"
    type="shortest_path_node"
    output="screen"
    clear_params="true" >

    <!-- Basic and control parameter -->
    <rosparam command="load" file="$(find mav_coverage_planning)/cfg/base_planner.yaml" />
    <!-- Polygon, altitude, and world frame ID -->
    <rosparam command="load" file="$(find mav_coverage_planning)/cfg/example_polygon.yaml" />
  </node>

</launch>
